sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/ApiClientTest.java

```diff
@@ -86,6 +88,11 @@ public class ApiClientTest {
 
   private ClientTestApp app;
 
+  @BeforeAll
+  static void beforeAll() {
+    WireMock.configureFor("http", "localhost", WIRE.getPort());
```

Comment:
Do we need a comment here? At least I don't understand why this configuration is needed.